src/OpenTelemetry.Api/OpenTelemetry.Api.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
```

Comment:
I believe we may hit some issue by changing target FW for Api. @austinlparker you mentioned OpenTelemetry bridge may cause issues.

Reply:
Austin mentioned the OpenTracing:

https://github.com/opentracing/opentracing-csharp/blob/master/src/OpenTracing/OpenTracing.csproj#L13

Reply:
Maybe it will be easier to update opentracing rather than use unsupported by Microsoft version of .net framework?